testing-common/testing-common.gradle

```diff
@@ -32,7 +32,7 @@ dependencies {
 
   api deps.groovy
 
-  testImplementation project(':instrumentation-api')
+  api project(':instrumentation-api')
```

Comment:
Thanks. This is indeed the wrong place for this. Changed it back. Perhaps because I also had an issue with logging I automatically did a similar change here. The issue is again that I have a common module that is used by 2 other modules where muzzle check failed during runtime because it couldn't find `io.opentelemetry.instrumentation.api.tracer.BaseTracer` presumably because it wasn't registered as helper class during compile (idk might be something completely different). I changed common module from javaagent -> library which has `api project(':instrumentation-api')`. I hope this was the correct choice.